app/views/layout.njk

```diff
@@ -1,5 +1,6 @@
 {% extends "govuk/template.njk" %}
 {% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+{% from "govuk/components/header/macro.njk" import govukHeader %}
```

Comment:
We can actually get rid of this.
Just realised this - as it gets automatically included with the layout.